destination/writer/errors.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writer
+
+import "errors"
+
+var (
+	// errEmptyPayload occurs when there's no payload to insert or update.
```

Comment:
errEmptyPayload => errNoPayload

Reply:
But it is an `errNoPayload`:
https://github.com/conduitio-labs/conduit-connector-clickhouse/blob/destination/destination/writer/errors.go#L21

Reply:
I think Maks means that the comment to the error doesn't match the error name.

Reply:
Oh, ok. 👍